vcd/resource_vcd_vapp.go

```diff
@@ -577,3 +652,39 @@ func tryUndeploy(vapp govcd.VApp) error {
 	}
 	return nil
 }
+
+// resourceVcdVappImport is responsible for importing the resource.
+// The following steps happen as part of import
+// 1. The user supplies `terraform import _resource_name_ _the_id_string_` command
+// 2. `_the_id_string_` contains a dot formatted path to resource as in the example below
+// 3. The functions splits the dot-formatted path and tries to lookup the object
+// 4. If the lookup succeeds it sets the ID field for `_resource_name_` resource in statefile
+// (the resource must be already defined in .tf config otherwise `terraform import` will complain)
+// 5. `terraform refresh` is being implicitly launched. The Read method looks up all other fields
+// based on the known ID of object.
+//
+// Example resource name (_resource_name_): vcd_vapp.my-vapp
+// Example import path (_the_id_string_): org.vdc.my-vapp
```

Comment:
By looking at this, and the above changes to use ID, it's not clear whether this is expecting name or ID. I suggest we write `org-name.vdc-name.my-vapp-name` or something like that.

Reply:
Nobody expects to use a resource ID explicitly. After 12 data sources in this release this concept should have been clear by now.

IDs are always generated and not provided manually in the scripts. I will add `-name` although I think it's unnecessary

Reply:
@Didainius you have indicated a gravitation to use IDs, do you have any import where you require or will require ID soon?